### ISSUE TITLE
Uncaught ReferenceError: performanceNow is not defined

### ISSUE THREAD

**Issue by wietsedevries** [facebook/react-devtools#651]
After hours of pulling my hair I found that React developer tools causes a referenceError on sites using Relay.  [`Check example`](http://facebook.github.io/relay/prototyping/playground.html#/) No idea why, but haven't experienced this issue before today. 

![Image of error](http://wietsedevries.eu/assets/images/relay.png)

**Comment by ethulia** [comment]
This is affecting us too. No one with react dev tools installed can use our site :(

**Comment by Rodeoclash** [comment]
Confirm that I'm getting this as well, looks like something broke in the last release.

Like @wietsedevries I'm using Relay so perhaps related to that.

**Comment by timsuchanek** [comment]
Same here. We're using Relay `0.9.3`. Also no one with dev tools can use our app anymore :/

**Comment by gaearon** [comment]
I'm sorry about that, I screwed up. The issue was fixed in latest update (released about 30 hours ago) so the issue will go away when people's addon gets updated. Chrome does this automatically.

**Comment by gaearon** [comment]
Can you verify that your sites work with React DevTools 2.1.3 (latest version)?

**Comment by gaearon** [comment]
Going to close this as I assume they do, but let me know if not!

**Comment by timsuchanek** [comment]
@gaearon thanks for the fast reaction! It is indeed fixed in 2.1.3.
In the meantime until all plugins are updated, we btw use this small workaround:
https://github.com/csnover/TraceKit
and
```js
TraceKit.report.subscribe((error) => {
  if (error.message === 'performanceNow is not defined') {
    alert(
      'The console cannot start because you are using React Devtools, '
      + 'which have a problem with Relay at the moment. '
      + 'Please disable React Devtools. If that doesn\'t help, please '
      + 'consult us in Slack. Thanks for your understanding.',
    )
  }
})
```
